style.css

```diff
@@ -4,34 +4,35 @@ body {
   color: #333;
   font-family: Lato, sans-serif;
 }
-.aaa {
+.description-method {
   width: 500px;
   margin: 0 auto;
   display: block;
   text-align: right;
 }
-.aaa img {
+.description-method__img {
   width: 100%;
 }
-.aaa .more_info {
+.description-method__link-more-info {
   font-family: fantasy, cursive;
 }
 
 @media (max-width:768px) {  
-  .aaa { text-align: center;
+  .description-method { text-align: center;
 }
 }
-.centered-main-page-element {
+.centered-main-page {
   display: block;
   width: 500px;
   margin: 0 auto 0;
 }
-.task {
+.add-item__input-new-task_stretch-field,
+.task_stretch-field {
   width: 56%;
   display: inline-block;
   flex-grow: 1;
 }
-.task-row-wrapper {
+.add-item__task-row-wrapper {
   display: flex;
 }
 ul {
```

Comment:
`body`, `ul`, `li`, `input` and `button` tag selectors doesn't match `basic-3.4` rule